src/pages/workspaces/workspace/WorkspaceContainer.ts

```diff
@@ -379,14 +466,30 @@ const useAppPolling = (workspace) => {
   return { apps, refreshApps };
 };
 
+interface WrapWorkspaceProps {
+  breadcrumbs: (props: { name: string; namespace: string }) => ReactNode;
+  activeTab?: string;
+  title: string;
+}
+
+interface WrappedComponentProps extends ComponentPropsWithRef<any> {
+  workspace: Workspace;
+  refreshWorkspace: () => void;
+  analysesData: AppDetails & CloudEnvironmentDetails;
+  storageDetails: StorageDetails;
+}
+
 export const wrapWorkspace =
-  ({ breadcrumbs, activeTab, title, topBarContent, showTabBar = true }) =>
```

Comment:
For every usage of this, neither topBarContent nor showTabBar are ever provided.

Reply:
Yeah, I think I eliminated the usage of topBarContent because having the workspace menu and workspace state within the tab component made no sense semantically and caused a11y issues.